jwt.js

```diff
@@ -25,7 +25,7 @@ function wrapStaticSecretInCallback (secret) {
   }
 }
 
-function fastifyJwt (fastify, options, next) {
+async function fastifyJwt (fastify, options, next) {
```

Comment:
it's better to avoid mixing async/await and callback code. this code is already callback based and we can't change it, therefore we need to be able to handle the promise without making this function async. you'll find plenty of examples of that in various codebases. fastify itself is one of them since it supports many function signatures both as callback and as promises

Reply:
That makes sense. I dropped the async implementation